### ISSUE TITLE
Screen Reader: Many Pages: Buttons with option/value announced without visible label

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77564]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using iOS VoiceOver or Android TalkBack, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the menu buttons.
5. Note how the controls are announced

Other Occurrences:
1. Profile > Status: the "Clear after" button with option/value selected
2. Profile > Timezone: the "Timezone" button with option/value selected
3. Profile > Address: the "Country" and "State" buttons with option/value selected
4. Preferences and subsections: "Priority mode", "Language", "Payment currency" and "Theme" buttons with option/value selected
5. Add bank > What are your account details?: the "Currency" button with option selected
6. Add bank > Check your info: all buttons with option/value selected
7. Create Expense > Confirm Details: all buttons with option/value selected
8. Group Chat > Details: the "Group name" button selected
9. Group Chat > Details > Settings: the "Notify me about new messages" button with option selected
10. Group Chat > Details > Private notes: the "My note" button with value selected
11. Add bank > What are your account details?: the "Currency" button with option selected
12. Add bank > Check your info: all buttons with option/value selected
13. Reports > Filters: all buttons with option/value selected
14. Create workspace > Reports: the "Default report title" with value selected
15. Create workspace > Confirm Workspace: the "Default currency" button
16. Create workspace > Overview: all buttons with option/value selected

## Expected Result:
Screen readers should announce both the label and the option/value associated with the button, e.g.:
- 'Display name: <name>'
- 'Timezone: <option>'

## Actual Result:
The buttons with chevrons are announced without their programmatic name:
- name
- option

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Update the current accessibilityLabel to include both - the name and option, or remove it altogether to announce the visible text.

https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2430" alt="Image" src="https://github.com/user-attachments/assets/0162a665-1c3d-4799-a632-6bd5fb57d0a7" />

<img width="1179" height="3575" alt="Image" src="https://github.com/user-attachments/assets/b5fc0707-b412-4954-b01d-6cdde50144ed" />

https://github.com/user-attachments/assets/429b7b81-d405-41f2-b537-5e3e6636fb6c

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005942
**Issue reported by:** NA
**Slack conversation:** NA
